Mathlib/Algebra/Group/End.lean

```diff
@@ -806,18 +806,18 @@ def conj [AddGroup G] : G →+ Additive (AddAut G) where
     rfl
 
 @[simp]
-theorem conj_apply [AddGroup G] (g h : G) : conj g h = g + h + -g :=
+theorem conj_apply [AddGroup G] (g h : G) : (conj g).toMul h = g + h + -g :=
```

Comment:
Strictly speaking, this one was ok, as the `toMul` was inserted automatically (check doc-gen)!

Either way, this spelling is less surprising.